sdk/apis/third_party/conditions/apis/conditions/v1alpha1/register.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2024 The Kube Bind Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// SchemeGroupVersion is group version used to register these objects
+var SchemeGroupVersion = schema.GroupVersion{Group: "conditions.kcp.io", Version: "v1alpha1"}
```

Comment:
what is this for?

Reply:
kcp SDK has this generated for `applyconfiguration` 
```
v1alpha1 "github.com/kube-bind/kube-bind/sdk/apis/third_party/conditions/apis/conditions/v1alpha1"

// Group=conditions, Version=v1alpha1
case v1alpha1.SchemeGroupVersion.WithKind("Condition"):
```

And we added this into kcp at some point. Just adding this here too.

Reply:
Ah! kcp.io group. I think its artificial, so we could make it kube-bind.io. But this is kcp SDK spefic

Reply:
https://github.com/kube-bind/kube-bind/blob/main/sdk/kcp/applyconfiguration/utils.go#L41